Vagrantfile

```diff
@@ -25,7 +25,15 @@
 # each plugin directory is that "reddit-plugin-NAME" should be in the directory
 # {ROOTDIR}/NAME.
 #
-# To start your vagrant box simply enter `vagrant up` from {ROOTDIR}/reddit.
+# This VagrantFile allows for the creation of two VMs:
+#   * default: the primary VM, with all services necessary to run reddit
+#              locally against the local codebase.
+#   * travis:  Testing-only VM suitable for running `nosetests` and debugging
+#              issues encountered without having to wait for travis-ci to pick
+#              up the build.  This will *not* be the same environment as
+#              travis, but it should be useful for repairing broken tests.
+#
+# To start your vagrant box simply enter `vagrant up default` from {ROOTDIR}/reddit.
```

Comment:
On closer reading of the [Vagrant docs](https://docs.vagrantup.com/v2/multi-machine/) it looks like yes.  [Unfortunately they don't say as much until several sections after they say "Commands that only make sense to target a single machine, such as vagrant ssh, now *require* the name of the machine to control." which initially threw me.]